src/dynatrace/opentelemetry/metrics/export/serializer.py

```diff
@@ -67,26 +67,30 @@ class DynatraceMetricsSerializer:
 
     def __init__(
         self,
-        prefix: Optional[str],
-        dimensions: Optional[Mapping],
+        prefix: Optional[str] = "",
+        default_dimensions: Optional[Mapping] = None,
+        one_agent_dimensions: Optional[Mapping] = None,
     ):
         self._prefix = prefix
         self._is_delta_export = None
 
-        self._dimensions = self._normalize_dimensions(dimensions)
+        self._default_dimensions = self._normalize_dimensions(
+            default_dimensions)
+        self._one_agent_dimensions = self._normalize_dimensions(
+            one_agent_dimensions)
 
     @classmethod
     def _normalize_dimensions(cls, dimensions):
         dim_dict = {}
         if dimensions:
-            # normalize the dimensions once, so it doesnt have to be repeated
-            # for every serialisation
+            # normalize the dimensions once, so it doesn't have to be repeated
+            # for every serialization
             for k, v in dimensions.items():
                 key = cls._normalize_dimension_key(k)
                 if key:
                     dim_dict[key] = cls._normalize_dimension_value(v)
 
-        return dim_dict
+        return [(k, v) for k, v in dim_dict.items()]
```

Comment:
Why was this changed?

Reply:
i think this was due to an oversight on my part, thinking that it wasn't possible to type-hint `dict[str, str]` in the method signature of `_make_unique`. I since figured out a way to do that and moved the code here back to using dictionaries.